src/components/Buttons/Button.tsx

```diff
@@ -0,0 +1,13 @@
+import styles from "./Button.module.css";
+import "../../globals.css";
+
+type ButtonProps = {
+  size: string;
```

Comment:
You could make sure that you don't add a not-existing size

```suggestion
  size: 'normal' | 'wide' | 'small';
```